packs/aws/config.yaml

```diff
@@ -1,8 +1,8 @@
 ---
 setup:
-  region: ""
-  aws_access_key_id: ""
-  aws_secret_access_key: ""
+  region: "us-east-1"
```

Comment:
In my view safer to not have a value specified and have anyone using the pack make an explicit choice.